### DISCUSSION TITLE
How to customize the online/exited job message

### DISCUSSION THREAD

**Discussion by diegomdrs** [breejs/bree#209] [Q&A]
Hi,

Is there any way to customize the online/exited job message from:

~~~
'Worker for job <name> online' 
'Worker for job <name> exited with code 0'
~~~

For example, to:

~~~
'[<name>] Worker for job online' 
'[<name>] Worker for job exited with code 0'
~~~

**Comment by Romax-91** [comment]
No way. This is given by a constant.

https://github.com/breejs/bree/blob/4f388fe70c1c27293ecff0bd6c8f8bd8a649405b/src/index.js#L391
